libs/chain-signatures/contract/src/lib.rs

```diff
@@ -1816,7 +1804,7 @@ mod tests {
         let threshold = Threshold::new(3);
 
         // Submit valid attestations for first 3 participants
-        submit_valid_attestations(&mut contract, &participants, &[0, 1, 2]);
+        submit_valid_attestations(&mut contract, &participants, &[0, 1, 2, 3]);
```

Comment:
You should probably update the comment for this test, since `TeeQuoteStatus::None` no longer exists.